.gitignore

```diff
@@ -15,7 +15,6 @@ build
 captures
 .externalNativeBuild
 .lastAutoPublishContentsHash
-# Not sure if we should ignore those, but we don't check them
-# into Application Services
```

Comment:
AFAICT we do actually commit these in both appservices and a-c, and they're necessary in order for the `./gradlew` wrapper script to work.